sitewit/services.py

```diff
@@ -0,0 +1,78 @@
+import base64
+import logging
+
+from demands import HTTPServiceClient, HTTPServiceError
+from yoconfig import get_config
+
+log = logging.getLogger(__name__)
+
+
+class SitewitService(HTTPServiceClient):
+    """
+    Client for SiteWit's API::
+
+        sitewitservice = SitewitService()
+        response = sitewitservice.get_account(account_token).
+
+    """
+
+    DEFAULT_URL = 'https://sandboxpapi.sitewit.com'
```

Comment:
It does, but currently we don't have this "url" in config, so we have a default.